src/main.cpp

```diff
@@ -69,6 +75,15 @@ int main(int argc, char* args[]) {
 
         // UI
 
+        // Maintain a maximum frame rate of 60fps
+        auto elapsed_time = std::chrono::steady_clock::now() - start_time;
+        auto elapsed_frames = std::chrono::duration_cast<FrameDuration>(elapsed_time);
```

Comment:
Can we just calculate the time left and just call `usleep(time_left)`? I think it'll be simpler and save CPU cycles.